### PR TITLE
Stage 0.35.0 release

### PR DESCRIPTION
**Pull request by jpogran**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -77,6 +128,16 @@ INTERNAL:
...
+## 0.34.3 (03 September 2024)
+
+ENHANCEMENTS:
+
+* Generate built in providers ([#1817](https://github.com/hashicorp/terraform-ls/issues/1817))
+
+INTERNAL:
+
+* Add artifacts manifest ([#1784](https://github.com/hashicorp/terraform-ls/issues/1784))
+
```
Comment (jpogran) [inline CHANGELOG.md:140]: It looks like the 0.34.3 was misplaced when I did the last release. Unfortunately changie orders on version number, not date, so this looks out of order.

File: .changes/v0.35.0.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+## 0.35.0 (15 October 2024)
+
+NOTES:
+
+This release adds support for Terraform Stack and Deployment files. This provides intelligent completion for Terraform Stacks blocks and attributes in Stack and Deployment files, including suggesting only valid completions for variables in component and deplyoment blocks. Hover documentation for all Stack and Deploy blocks, with helpful type and usage information. Early Validation has been extended to produce diagnostics along with syntax validation for Stack and Deployment files. Reference support allows code navigation in and between Stack and Deploy files. This also provides formatting support for both Stack and Deploy files.
```
Comment (ansgarm) [inline .changes/v0.35.0.md:5]:
```suggestion
This release adds support for Terraform Stack and Deployment files. This provides intelligent completion for Terraform Stacks blocks and attributes in Stack and Deployment files, including suggesting only valid completions for variables in component and deployment blocks. Hover documentation for all Stack and Deploy blocks, with helpful type and usage information. Early Validation has been extended to produce diagnostics along with syntax validation for Stack and Deployment files. Reference support allows code navigation in and between Stack and Deploy files. This also provides formatting support for both Stack and Deploy files.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,59 @@
-## 0.34.3 (03 September 2024)
+## 0.35.0 (15 October 2024)
+
+NOTES:
+
+This release adds support for Terraform Stack and Deployment files. This provides intelligent completion for Terraform Stacks blocks and attributes in Stack and Deployment files, including suggesting only valid completions for variables in component and deplyoment blocks. Hover documentation for all Stack and Deploy blocks, with helpful type and usage information. Early Validation has been extended to produce diagnostics along with syntax validation for Stack and Deployment files. Reference support allows code navigation in and between Stack and Deploy files. This also provides formatting support for both Stack and Deploy files.
```
Comment (ansgarm) [inline CHANGELOG.md:5]:
```suggestion
This release adds support for Terraform Stack and Deployment files. This provides intelligent completion for Terraform Stacks blocks and attributes in Stack and Deployment files, including suggesting only valid completions for variables in component and deployment blocks. Hover documentation for all Stack and Deploy blocks, with helpful type and usage information. Early Validation has been extended to produce diagnostics along with syntax validation for Stack and Deployment files. Reference support allows code navigation in and between Stack and Deploy files. This also provides formatting support for both Stack and Deploy files.
```

File: .changes/v0.35.0.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+* Enable component references ([terraform-schema#386](https://github.com/hashicorp/terraform-schema/issues/386))
+* Enable ephemeral values for variable ([terraform-schema#387](https://github.com/hashicorp/terraform-schema/issues/387))
+* Enable output references ([terraform-schema#384](https://github.com/hashicorp/terraform-schema/issues/384))
+* Enable provider references ([terraform-schema#385](https://github.com/hashicorp/terraform-schema/issues/385))
+* Add Address to variable block schema for stacks to enable references ([terraform-schema#383](https://github.com/hashicorp/terraform-schema/issues/383))
+* Add deployments store block schema ([terraform-schema#382](https://github.com/hashicorp/terraform-schema/issues/382))
+* Add input block schema and deprecate variable block ([terraform-schema#381](https://github.com/hashicorp/terraform-schema/issues/381))
+* Validate Stack and Deployment files for unreferenced origins ([#1797](https://github.com/hashicorp/terraform-ls/issues/1797))
+* Early decode deployment config to support references to store blocks ([terraform-schema#390](https://github.com/hashicorp/terraform-schema/issues/390))ncbtbenihcbbvctiviuekvbhdfbeegbrtbbivt
```
Comment (ansgarm) [inline .changes/v0.35.0.md:30]:
```suggestion
* Early decode deployment config to support references to store blocks ([terraform-schema#390](https://github.com/hashicorp/terraform-schema/issues/390))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,8 +1,59 @@
...
+* Enable component references ([terraform-schema#386](https://github.com/hashicorp/terraform-schema/issues/386))
+* Enable ephemeral values for variable ([terraform-schema#387](https://github.com/hashicorp/terraform-schema/issues/387))
+* Enable output references ([terraform-schema#384](https://github.com/hashicorp/terraform-schema/issues/384))
+* Enable provider references ([terraform-schema#385](https://github.com/hashicorp/terraform-schema/issues/385))
+* Add Address to variable block schema for stacks to enable references ([terraform-schema#383](https://github.com/hashicorp/terraform-schema/issues/383))
+* Add deployments store block schema ([terraform-schema#382](https://github.com/hashicorp/terraform-schema/issues/382))
+* Add input block schema and deprecate variable block ([terraform-schema#381](https://github.com/hashicorp/terraform-schema/issues/381))
+* Validate Stack and Deployment files for unreferenced origins ([#1797](https://github.com/hashicorp/terraform-ls/issues/1797))
+* Early decode deployment config to support references to store blocks ([terraform-schema#390](https://github.com/hashicorp/terraform-schema/issues/390))ncbtbenihcbbvctiviuekvbhdfbeegbrtbbivt
```
Comment (ansgarm) [inline CHANGELOG.md:30]:
```suggestion
* Early decode deployment config to support references to store blocks ([terraform-schema#390](https://github.com/hashicorp/terraform-schema/issues/390))
```

**Review by ansgarm — APPROVED** [review]
Looks good – just two small cleanup items

**Review by dbanck — APPROVED** [review]
Thank you for composing this

**Pull request merged**